src/org/daisy/dotify/api/formatter/TableOfContents.java

```diff
@@ -21,4 +21,23 @@
 	 */
 	public void endEntry();
 	
+    /**
+	 * Starts a new entry-on-resumed with the supplied properties.
+	 * 
+	 * @param range the range of this entry on resumed
+	 */
+	public void startEntryOnResumed(String range);
```

Comment:
Maybe it would be nice to have a parsed range object in the API instead of the String?

Reply:
In other words, I would move the `TocEntryOnResumedRange` class from the other PR to the API. And I would also move the range parsing code from `TableOfContentsImpl` to `TocEntryOnResumedRange` itself.

Reply:
Done.